indico/modules/attachments/controllers/management/base.py

```diff
@@ -75,8 +75,11 @@ class ManageAttachmentsMixin:
     def _process(self):
         tpl_args = {'linked_object': self.object, 'linked_object_type': self.object_type,
                     'attachments': get_attached_items(self.object)}
-        if self.object_type in ('event', 'category'):
+        if self.object_type == 'event':
             return self.wp.render_template('attachments.html', self._target, **tpl_args)
+        elif self.object_type == 'category':
+            tpl_args['category'] = self._target
```

Comment:
why not `category=self._target` in the next line before `**tpl_args`? Also, I would really prefer to not use `self._target` for new stuff but just set `self.category` in the base RH for anything category related